### PR TITLE
Reviewlist refactor

### PR DESCRIPTION
**Pull request by ggbbi**
Here, I focused on reorganizing the components in `ReviewListContainer` and refactoring them to clean up the conditional rendering. I got rid of `if` statements within components and I'm using inline ternaries now instead. The Sept. 20 (more recent half of) commits are the relevant ones, since this branch comes off of my work on 1.2.4 (not yet pushed).

### CODE REVIEW

File: client/src/Components/ReviewsContainer/Breakdowns/RatingAvgs.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  let total = 0
+  let count = 0
+  let average = 0
+
+  for (let rating in reviewMetadata.ratings) {
+    total += rating * reviewMetadata.ratings[rating]
+    count += 1 * reviewMetadata.ratings[rating]
+  }
+  if (total > 0) {
```
Comment (spencerlepine) [inline client/src/Components/ReviewsContainer/Breakdowns/RatingAvgs.jsx:18]: is average being stored in state later on?

File: client/src/Components/ReviewsContainer/ReviewsContainer.jsx [hunk 2]
```diff
@@ -1,13 +1,25 @@
 import React from "react"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
 import useClickLogger from "../../hooks/useClickLogger.jsx"
-import ReviewList from "./List/ReviewList.jsx"
+import Breakdowns from "./Breakdowns/Breakdowns.jsx"
+import List from "./List/List.jsx"
 import "bootstrap/dist/css/bootstrap.min.css"
```
Comment (spencerlepine) [inline client/src/Components/ReviewsContainer/ReviewsContainer.jsx:8]: we should be able to have one import in index.jsx.

```js
// src/index.jsx
import React from "react"
import "bootstrap/dist/css/bootstrap.min.css"

...
ReactDOM.render(<Component />)
```

**Review by spencerlepine — APPROVED** [review]
Looks good, nothing to refactor.

Little confused about `RatingBar.jsx` count variable, but it looks like it will render the right values.
Another note is that we can probably use 1 import for the bootstrap css file (ReviewsContainer.jsx)

File: client/src/Components/ReviewsContainer/ReviewsContainer.jsx (hunk 2 above)
Comment (ggbbi) [reply client/src/Components/ReviewsContainer/ReviewsContainer.jsx:8]: will bring this up in team meeting

**Pull request merged**